sets.json

```diff
@@ -1125,61 +710,6 @@
       "White_Dune_-_Mui_Ne.y4m"
     ]
   },
-  "subset1-64": {
```

Comment:
Same for this one.

Reply:
As above

Reply:
It was used for debugging frame edge issues on rav1e. I can also upload it.